TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java

```diff
@@ -24,11 +25,13 @@
 
 public class InsertData extends AppCompatActivity {
 
-    String id,name,poojaTyp,overall,money,paidCheck ="NOT PAID";
+    String id,name,poojaTyp,overall,money,paidCheck ="NOT PAID", Amount;
```

Comment:
Avoid starting variable names with capital letters. Please give it a different name starting with a lowercase character.